docs/meeting-notes.rst

```diff
@@ -14,8 +14,8 @@ the `meeting notes here <https://docs.google.com/document/d/e/2PACX-1vRerhoxG-wO
 
 **Schedule**
 
-* Early Weeks: 4p GMT
-* Late Weeks: 8p GMT
```

Comment:
Just throwing this out there. Do we need two times still? We originally were optimizing for Pacific and UK timezones. I would think the 12pm ET would work for both at this point.

Reply:
I think this is part of a broader conversation about what is the point of these meetings and who we want to engage.